src/styles/card.css

```
@@ -0,0 +1,81 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+}
+
+.card__picture {
+  width: 160px;
+  height: 134px;
+  align-self: center;
+}
+
+.card__product-name {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: var(--main-color);
+  margin-top: 40px;
+}
+
+.card__product-code {
+  font-size: 10px;
+  color: var(--secondary-color);
+  margin-top: 4px;
+}
+
+.card__rating-panel {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
```

```suggestion
  align-items: center;
```

maybe this code is unnecessary